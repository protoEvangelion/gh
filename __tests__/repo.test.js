/**
 * Â© 2013 Liferay, Inc. <https://liferay.com> and Node GH contributors
 * (see file: CONTRIBUTORS)
 * SPDX-License-Identifier: BSD-3-Clause
 */

'use strict'

const { runCmd } = require('./testUtils')

describe('E2E: Repo Module Test', () => {
    it('List Repos `gh re --list`', done => {
        expect(runCmd('bin/gh.js re --list')).toMatchSnapshot()
        done()
    })

    it('Create new repo `gh re --new foo --init`', done => {
        expect(runCmd('bin/gh.js re --new foo --init')).toMatchSnapshot()
        done()
    })

    it('Fork a repo `gh re --fork prettier --user prettier`', done => {
        expect(runCmd('bin/gh.js re --fork prettier --user prettier')).toMatchSnapshot()
        done()
    })

    it('Delete repo `gh re --delete foo`', done => {
        expect(runCmd('bin/gh.js re --delete foo')).toMatchSnapshot()
        done()
    })
})
