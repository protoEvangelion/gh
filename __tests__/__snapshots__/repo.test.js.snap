// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`E2E: Repo Module Test Create new repo \`gh re --new foo --init\` 1`] = `
"Creating a new repo on protoEvangelion/foo
https://github.com/protoEvangelion/foo
[hook] gh re --browser  --user protoEvangelion --repo foo
"
`;

exports[`E2E: Repo Module Test Delete repo \`gh re --delete foo\` 1`] = `
"Deleting repo protoEvangelion/foo
"
`;

exports[`E2E: Repo Module Test Fork a repo \`gh re --fork prettier --user prettier\` 1`] = `
"Forking repo prettier/prettier on /prettier
https://github.com/protoEvangelion/prettier
"
`;

exports[`E2E: Repo Module Test List Repos \`gh re --list\` 1`] = `
"protoEvangelion/100-days-of-code
protoEvangelion/adopt-a-family
protoEvangelion/alloy-ui
"
`;
